MiniApp/Classes/core/Protocols/MiniAppMessageDelegate.swift

```diff
@@ -14,26 +11,12 @@ public protocol MiniAppMessageDelegate: MiniAppUserInfoDelegate, MiniAppShareCon
     /// result (allowed/denied) should be returned.
     func requestDevicePermission(permissionType: MiniAppDevicePermissionType, completionHandler: @escaping (Result<MASDKPermissionResponse, MASDKPermissionError>) -> Void)
 
-    /// Interface that should be implemented in the host app that handles the code to request any permission and the
-    /// result (allowed/denied) should be returned.
-    @available(*, deprecated,
-        message: "Since version 2.8.0, this method is deprecated",
-        renamed: "requestDevicePermission(permissionType:completionHandler:)")
-    func requestPermission(permissionType: MiniAppPermissionType, completionHandler: @escaping (Result<MASDKPermissionResponse, MASDKPermissionError>) -> Void)
```

Comment:
I think there was also a default implementation for this which needs to be refactored/removed. See this PR: 
https://github.com/rakutentech/ios-miniapp/pull/200/files#diff-607bf474e1963f1a8e4c576602075f15b9927f34ccb84af201e017af01bfa0daR62

Reply:
oh, I'm sorry, that's really strange they did not appear in my global search and XCode did not pop a warning :-/
I'll fix this 🤔